src/setup.ts

```diff
@@ -19,6 +19,8 @@ import {verifyChecksum} from "./checksum"
     var api_url = `https://${env}.api.stepsecurity.io/v1`;
     var web_url = "https://app.stepsecurity.io";
 
+    let token = core.getInput('token');
```

Comment:
@Devils-Knight please add a comment that this is to avoid rate limit

Reply:
@varunsh-coder added comment and updated the inputs for `action.yml`.